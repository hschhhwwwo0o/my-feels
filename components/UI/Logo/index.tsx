import React, { FunctionComponent } from "react";

const Logo: FunctionComponent = () => {
  return (
    <>
      <div>
        <h1 className="text-3xl">
          <i>my </i>
          <span className="font-bold">feels</span>
        </h1>
      </div>
    </>
  );
};

export default Logo;
