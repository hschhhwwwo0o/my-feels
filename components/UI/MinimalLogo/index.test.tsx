import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import Logo from "./index";

test("Test NoteCard component", () => {
  render(<Logo />);
  const el: HTMLElement = screen.getByRole("banner");
  expect(el).toBeInTheDocument();
  expect(el).toMatchSnapshot();
});
