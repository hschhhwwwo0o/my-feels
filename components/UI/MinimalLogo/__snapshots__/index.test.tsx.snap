// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test NoteCard component 1`] = `
<a
  href="/"
  role="banner"
>
  <div
    class="cursor-pointer"
  >
    <h1
      class="text-lg dark:text-white transition-all duration-1000"
    >
      <i>
        m
      </i>
      <span
        class="font-bold"
      >
        f
      </span>
    </h1>
  </div>
</a>
`;
